python/sandbox/dialects/_linalg_transform_ops_ext.py

```diff
@@ -336,3 +338,70 @@ class SequenceOp:
   def __init__(self, *, loc=None, ip=None):
     super().__init__(loc=loc, ip=ip)
     self.body.blocks.append()
+
+
+##===----------------------------------------------------------------------===##
+## LinalgExt specific transforms
+##===----------------------------------------------------------------------===##
+
+
+class TileToLinalgExtTileOp:
+  """Specialization for the TileToLinalgExtTileOp class."""
+
+  def __init__(self,
+               target: Union[ir.Value, ir.Operation, ir.OpView],
+               *,
+               sizes: IntListArg = None,
+               loc=None,
+               ip=None):
+    sizes = _ensure_array_attr(sizes, [])
+    operation_type = pdl.OperationType.get()
+    super().__init__(operation_type, target, sizes, loc=loc, ip=ip)
```

Comment:
_ensure_array_attr(sizes, [])?

Reply:
if you are suggesting inlining the expression, I kept it outlined for consistency with the rest.